src/pages/Compare/index.spec.tsx

```diff
@@ -260,7 +261,10 @@ describe(__filename, () => {
 
     const root = render({ store });
 
-    expect(root.find(`.${styles.error}`)).toHaveLength(2);
+    expect(
+      getContentShellPanel(root, 'mainSidePanel').find(`.${styles.error}`),
```

Comment:
Shouldn't this error be displayed in the `Header`, rather than in the `mainSidePanel`?

Reply:
This component is far removed from where the `Header` is rendered so it would be tricky to do that. Also, I think it's still readable to see an error in this panel.